src/components/OrderReport/OrderReport.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Avatar } from "../Avatar/Avatar";
+import imgOne from "../../assets/Avatar/1.png";
+import imgTwo from "../../assets/Avatar/2.png";
+import imgThree from "../../assets/Avatar/3.png";
+import imgFour from "../../assets/Avatar/4.png";
+import imgFive from "../../assets/Avatar/5.png";
+import "./OrderReport.scss";
+
+const OrderReport = () => {
+  const orders = [
+    {
+      avatar: <Avatar bgColor={"red"} avatarImg={imgOne} />,
+      customer: "Eren Jaegar",
+      menu: "Spicy seasoned seafood noodles",
+      totalPayment: "$125",
+      status: "Completed",
+    },
```

Comment:
Buradaki data proje genelinde datanın aynı yerden gelmesi için db.js içerisinde tanımlanmalı. Oradan avatar propslarını data içerisine tanımlayarak bu sayfada bu şekilde kullanabilirsin.
`<Avatar bgColor={order.avatarColor}  avatarImg={order.avatarImg}  />`

db.js içerisinde örnek tanımlama:
```
{
    avatarImg: imgTwo,
    avatarColor: "yellow",
    customer: "Levi Ackerman",
    menu: "Beef dumpling in hot and sour soup",
    totalPayment: "$105",
    status: "Pending",
  },
```